File: frontend-main/src/models/rally/location.ts
import { LocationMarker } from "./locationMarker";

export class Location {
  id: string;
  title: string;
  locationMarkers: LocationMarker[];

  constructor(id: string, title: string, locationMarkers: LocationMarker[]) {
    this.id = id;
    this.title = title;
    this.locationMarkers = locationMarkers;
  }
}

File: flutter-application-main/lib/models/rally/location.dart
import 'package:dart_mappable/dart_mappable.dart';

import 'location_marker.dart';

@MappableClass()
class Location {
  final String id;
  final String title;
  final List<LocationMarker> locationMarkers;

  Location({
    required this.id,
    required this.title,
    required this.locationMarkers,
  });
}
